test/runtests.py

```diff
@@ -94,7 +102,7 @@
     test_root = os.path.realpath(args.override_test_root)
 
 # arch: x86, x64
-arch = 'x86' if args.x86 else ('x64' if args.x64 else None)
+arch = 'x86' if args.x86 else ('x64' if args.x64 else ('arm' if args.arm else ('arm64' if args.arm64 else None)))
```

Comment:
This... seems a bit ridiculous, especially with the two extra architectures. Could we make it into a dictionary lookup or a regular if tree?

Reply:
I see your point, this is changed to a series of if statements in the next commit. I don't know of a good way to do this that's not significantly bigger or harder to read, though.